src/projections/defaults.js

```diff
@@ -2,7 +2,7 @@ import { mapObject, isArray, assign } from '../utils';
 
 function createModel(config, ...additionalProps) {
   return assign({
-    key: config.key || null,
+    key: config.key || `default${config.tag || ''}`,
```

Comment:
The default key will duplicate.. I think it's the responsibility of projection-grid-vue/projection-grid-react to interpret the "null" key.